JavaErickHernandez/Day3/SortedList.java

```diff
@@ -0,0 +1,99 @@
+package bootcamp;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Scanner;
+
+public class SortedList {
+	static String numberOfInputs = "Enter the number of Students: ";
+	static String tryAgainMessage = "Try Again";
+	static String enterName = "Enter name: ";
+	static String enterId = "Enter id: ";
+	static String enterCGPA = "Enter CGPA: ";
+	static String error = "Error!";
+	static ArrayList<Student> students = new ArrayList<Student>();
```

Comment:
When you variable is a Collection, it is good practice put name in singular and them the the type, example: tableList, nameSet, etc...

Reply:
Thanks, I used to write it in plural so that i cant get confused with an object and the list, I will change it and keep it in mind